Arbor doesn't like being submoduled

Cmake generates a few files, which doesn't happen when you submodule Arbor. When you configure the submodule with Python bindings, Cmake errors:

```
pybind11 v2.5.0
 CMake Error at 3rd-party/arbor/python/CMakeLists.txt:74 (file):
   file COPY cannot find
   "/home/brent/sync/julich/code/arbor-gui/python/__init__.py": No such file
   or directory.

 CMake Error at 3rd-party/arbor/python/CMakeLists.txt:75 (file):
   file COPY cannot find "/home/brent/sync/julich/code/arbor-gui/VERSION": No
   such file or directory.
 Using X11 for window creation
 Configuring incomplete, errors occurred!
```

--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -71,8 +71,8 @@ add_subdirectory(test)
 # The module contains the dynamic library, __init__.py and VERSION information.
 set(python_mod_path "${CMAKE_CURRENT_BINARY_DIR}/arbor")
 set_target_properties(pyarb PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${python_mod_path}")
-file(COPY "${CMAKE_SOURCE_DIR}/python/__init__.py" DESTINATION "${python_mod_path}")
-file(COPY "${CMAKE_SOURCE_DIR}/VERSION" DESTINATION "${python_mod_path}")
+file(COPY "${PROJECT_SOURCE_DIR}/python/__init__.py" DESTINATION "${python_mod_path}")
+file(COPY "${PROJECT_SOURCE_DIR}/VERSION" DESTINATION "${python_mod_path}")
 
 # Determine the installation path, according to the Python version.
 # The installation for Python 3.8 would be:

